oml/datasets/images.py

```diff
@@ -162,8 +161,9 @@ def __len__(self) -> int:
         return len(self._paths)
 
     def visualize(self, item: int, color: TColor = BLACK) -> np.ndarray:
-        bbox = torch.tensor(self._bboxes[item]) if (self._bboxes is not None) else torch.tensor([torch.nan] * 4)
-        image = get_img_with_bbox(im_path=self._paths[item], bbox=bbox, color=color)
+        img = np.array(imread_pillow(self.read_bytes(self._paths[item])))
```

Comment:
you can read with `cv2` directly to `np`. In addition it's faster

Reply:
I did it to support paths as urls later on